packages/tocco-ui/src/Table/Table.scss

```diff
@@ -0,0 +1,31 @@
+.tocco-table {
+  table {
+    width: 100%;
+
+    thead tr th {
+      cursor: pointer;
+      /* stylelint-disable */
```

Comment:
Because of Stylelint rule property-no-vendor-prefix (https://stylelint.io/user-guide/rules/property-no-vendor-prefix/#property-no-vendor-prefix).

I intend to dispense all vendor prefixes in source files to keep them small and only to rely on well established properties. I prefer to add browser prefixes automatically by appropriate gulp or webpack plugins.